bench/rttanalysis: TestBenchmarkExpectation failed

bench/rttanalysis.TestBenchmarkExpectation [failed](https://tanzanite.cluster.engflow.com/invocation/536d5b62-1e8e-4866-ab73-4e5a22917fde) on master @ [22d9482e646ae127981386d77b7ebf729346fc3b](https://github.com/cockroachdb/cockroach/commits/22d9482e646ae127981386d77b7ebf729346fc3b):


```
    validate_benchmark_data.go:123: success: expected ORMQueries/pg_is_other_temp_schema_multiple_times to perform KV lookups in [6, 6], got 6
    validate_benchmark_data.go:123: success: expected ORMQueries/information_schema._pg_index_position to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:123: success: expected ORMQueries/hasura_column_descriptions to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:123: success: expected ORMQueries/hasura_column_descriptions_8_tables to perform KV lookups in [12, 12], got 12
    validate_benchmark_data.go:123: success: expected ORMQueries/hasura_column_descriptions_modified to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:123: success: expected ORMQueries/prisma_column_descriptions_updated to perform KV lookups in [3, 3], got 4
    validate_benchmark_data.go:123: success: expected ORMQueries/column_descriptions_json_agg to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:123: success: expected ORMQueries/npgsql_types to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:123: success: expected ORMQueries/npgsql_fields to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:123: success: expected ORMQueries/liquibase_migrations to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:123: success: expected ORMQueries/asyncpg_types to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:123: success: expected VirtualTableQueries/select_crdb_internal.tables_with_1_fk to perform KV lookups in [1, 1], got 1
    validate_benchmark_data.go:123: success: expected VirtualTableQueries/select_crdb_internal.invalid_objects_with_1_fk to perform KV lookups in [1, 1], got 1
    validate_benchmark_data.go:123: success: expected VirtualTableQueries/virtual_table_cache_with_schema_change to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected Revoke/revoke_all_on_1_table to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected ORMQueries/liquibase_migrations_on_multiple_dbs to perform KV lookups in [8, 8], got 8
    validate_benchmark_data.go:123: success: expected Truncate/truncate_2_column_2_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected AlterTableSplit/alter_table_split_at_1_value to perform KV lookups in [8, 8], got 8
    validate_benchmark_data.go:123: success: expected AlterTableSplit/alter_table_split_at_2_values to perform KV lookups in [11, 11], got 11
    validate_benchmark_data.go:120: fail: expected ORMQueries/prisma_column_descriptions to perform KV lookups in [133, 133], got 156
    validate_benchmark_data.go:123: success: expected Grant/grant_all_on_1_table to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected AlterTableDropConstraint/alter_table_drop_3_check_constraints to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected AlterTableAddCheckConstraint/alter_table_add_1_check_constraint to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected AlterTableAddCheckConstraint/alter_table_add_2_check_constraints to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected Truncate/truncate_1_column_0_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected Truncate/truncate_1_column_1_row to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected Truncate/truncate_1_column_2_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected Truncate/truncate_2_column_0_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected AlterTableAddForeignKey/alter_table_add_1_foreign_key to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:123: success: expected AlterTableAddForeignKey/alter_table_add_2_foreign_keys to perform KV lookups in [19, 19], got 19
    validate_benchmark_data.go:123: success: expected AlterTableAddForeignKey/alter_table_add_3_foreign_keys to perform KV lookups in [21, 21], got 21
    validate_benchmark_data.go:123: success: expected AlterTableAddForeignKey/alter_table_add_foreign_key_with_3_columns to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:123: success: expected AlterTableDropConstraint/alter_table_drop_1_check_constraint to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected AlterTableDropConstraint/alter_table_drop_2_check_constraints to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected Truncate/truncate_2_column_1_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected RevokeRole/revoke_1_role to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:123: success: expected Grant/grant_all_on_2_tables to perform KV lookups in [19, 19], got 19
    validate_benchmark_data.go:123: success: expected DropView/drop_2_views to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:76: see the -rewrite flag to re-run the benchmarks and adjust the expectations
    validate_benchmark_data.go:77: usage: ./dev test --rewrite pkg/bench/rttanalysis -f=TestBenchmarkExpectation/ --test-args '-rewrite-iterations=N'
    validate_benchmark_data.go:110: -- test log scope end --
test logs left over in: outputs.zip/logTestBenchmarkExpectation2195869141
--- FAIL: TestBenchmarkExpectation (1486.27s)
=== RUN   TestBenchmarkExpectation/ORMQueries
    --- FAIL: TestBenchmarkExpectation/ORMQueries (1313.86s)
=== RUN   TestBenchmarkExpectation/ORMQueries/prisma_column_descriptions
    test_server_shim.go:144: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    rtt_analysis_bench.go:206: ORMQueries/prisma_column_descriptions: got 156, expected 133
    rtt_analysis_bench.go:212: wrote jaeger trace to /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/testlogs/pkg/bench/rttanalysis/rttanalysis_test/shard_1_of_16_run_2_of_3/test.outputs/logTestBenchmarkExpectation2195869141/ORMQueries_prisma_column_descriptions.jaeger.json
        --- FAIL: TestBenchmarkExpectation/ORMQueries/prisma_column_descriptions (1313.86s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBenchmarkExpectation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-34288

bench/rttanalysis.TestBenchmarkExpectation [failed](https://tanzanite.cluster.engflow.com/invocation/c5c25279-aeaf-459c-8ea2-691092f678f7) on master @ [e747f6e6857a19d6048cb184b0f55c52cb8a6390](https://github.com/cockroachdb/cockroach/commits/e747f6e6857a19d6048cb184b0f55c52cb8a6390):


```
    validate_benchmark_data.go:123: success: expected ORMQueries/pg_attribute to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:123: success: expected ORMQueries/introspection_description_join to perform KV lookups in [4, 4], got 5
    validate_benchmark_data.go:123: success: expected ORMQueries/has_sequence_privilege to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:123: success: expected ORMQueries/has_schema_privilege to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:123: success: expected ORMQueries/has_table_privilege to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:123: success: expected ORMQueries/has_column_privilege_using_attnum to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:123: success: expected ORMQueries/pg_my_temp_schema to perform KV lookups in [3, 3], got 3
    validate_benchmark_data.go:123: success: expected ORMQueries/has_column_privilege_using_column_name to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:123: success: expected ORMQueries/pg_my_temp_schema_multiple_times to perform KV lookups in [3, 3], got 3
    validate_benchmark_data.go:123: success: expected ORMQueries/pg_is_other_temp_schema to perform KV lookups in [6, 6], got 6
    validate_benchmark_data.go:123: success: expected ORMQueries/pg_is_other_temp_schema_multiple_times to perform KV lookups in [6, 6], got 6
    validate_benchmark_data.go:123: success: expected ORMQueries/information_schema._pg_index_position to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:123: success: expected ORMQueries/hasura_column_descriptions_8_tables to perform KV lookups in [12, 12], got 12
    validate_benchmark_data.go:123: success: expected ORMQueries/hasura_column_descriptions_modified to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:123: success: expected ORMQueries/column_descriptions_json_agg to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:123: success: expected ORMQueries/prisma_column_descriptions_updated to perform KV lookups in [3, 3], got 4
    validate_benchmark_data.go:120: fail: expected ORMQueries/prisma_column_descriptions to perform KV lookups in [133, 133], got 154
    validate_benchmark_data.go:123: success: expected ORMQueries/npgsql_types to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:123: success: expected ORMQueries/npgsql_fields to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:123: success: expected ORMQueries/liquibase_migrations to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:123: success: expected Truncate/truncate_1_column_0_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected ORMQueries/asyncpg_types to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:123: success: expected Truncate/truncate_1_column_1_row to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected Truncate/truncate_1_column_2_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected Truncate/truncate_2_column_0_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected Truncate/truncate_2_column_1_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected AlterTableAddForeignKey/alter_table_add_1_foreign_key to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:123: success: expected AlterTableAddForeignKey/alter_table_add_2_foreign_keys to perform KV lookups in [19, 19], got 19
    validate_benchmark_data.go:123: success: expected AlterTableAddForeignKey/alter_table_add_3_foreign_keys to perform KV lookups in [21, 21], got 21
    validate_benchmark_data.go:123: success: expected AlterTableUnsplit/alter_table_unsplit_at_1_value to perform KV lookups in [7, 7], got 7
    validate_benchmark_data.go:123: success: expected AlterTableUnsplit/alter_table_unsplit_at_2_values to perform KV lookups in [9, 9], got 9
    validate_benchmark_data.go:123: success: expected DropDatabase/drop_database_0_tables to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:123: success: expected DropDatabase/drop_database_1_table to perform KV lookups in [18, 18], got 18
    validate_benchmark_data.go:123: success: expected DropDatabase/drop_database_2_tables to perform KV lookups in [18, 18], got 18
    validate_benchmark_data.go:123: success: expected AlterTableSplit/alter_table_split_at_1_value to perform KV lookups in [8, 8], got 8
    validate_benchmark_data.go:123: success: expected AlterTableSplit/alter_table_split_at_2_values to perform KV lookups in [11, 11], got 11
    validate_benchmark_data.go:123: success: expected ORMQueries/hasura_column_descriptions to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:123: success: expected RevokeRole/revoke_1_role to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:76: see the -rewrite flag to re-run the benchmarks and adjust the expectations
    validate_benchmark_data.go:77: usage: ./dev test --rewrite pkg/bench/rttanalysis -f=TestBenchmarkExpectation/ --test-args '-rewrite-iterations=N'
    validate_benchmark_data.go:110: -- test log scope end --
test logs left over in: outputs.zip/logTestBenchmarkExpectation4263179325
--- FAIL: TestBenchmarkExpectation (1427.49s)
--- FAIL: TestBenchmarkExpectation (1427.75s)
=== RUN   TestBenchmarkExpectation/ORMQueries/prisma_column_descriptions
    rtt_analysis_bench.go:206: ORMQueries/prisma_column_descriptions: got 154.5, expected 133
    rtt_analysis_bench.go:212: wrote jaeger trace to /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/testlogs/pkg/bench/rttanalysis/rttanalysis_test/shard_1_of_16_run_2_of_3/test.outputs/logTestBenchmarkExpectation4263179325/ORMQueries_prisma_column_descriptions.jaeger.json
        --- FAIL: TestBenchmarkExpectation/ORMQueries/prisma_column_descriptions (1200.70s)
=== RUN   TestBenchmarkExpectation/ORMQueries
    --- FAIL: TestBenchmarkExpectation/ORMQueries (1416.01s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBenchmarkExpectation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

bench/rttanalysis.TestBenchmarkExpectation [failed](https://tanzanite.cluster.engflow.com/invocation/9c730add-fd04-438b-8aa2-c4167e840476) on master @ [8f91e0324af0bd99cbfcc9d44ceeda45ad0960ec](https://github.com/cockroachdb/cockroach/commits/8f91e0324af0bd99cbfcc9d44ceeda45ad0960ec):


```
    validate_benchmark_data.go:123: success: expected Grant/grant_all_on_1_table to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected Grant/grant_all_on_2_tables to perform KV lookups in [19, 19], got 19
    validate_benchmark_data.go:123: success: expected AlterTableAddForeignKey/alter_table_add_1_foreign_key to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:123: success: expected AlterTableAddForeignKey/alter_table_add_2_foreign_keys to perform KV lookups in [19, 19], got 19
    validate_benchmark_data.go:123: success: expected AlterTableAddForeignKey/alter_table_add_3_foreign_keys to perform KV lookups in [21, 21], got 21
    validate_benchmark_data.go:123: success: expected AlterTableDropColumn/alter_table_drop_1_column to perform KV lookups in [18, 18], got 18
    validate_benchmark_data.go:123: success: expected AlterTableDropColumn/alter_table_drop_2_columns to perform KV lookups in [18, 18], got 18
    validate_benchmark_data.go:123: success: expected DropRole/drop_1_role to perform KV lookups in [34, 34], got 34
    validate_benchmark_data.go:123: success: expected DropRole/drop_2_roles to perform KV lookups in [43, 43], got 43
    validate_benchmark_data.go:123: success: expected GenerateObjects/generate_10_tables to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:123: success: expected GenerateObjects/generate_1000_tables_-_this_test_should_use_the_same_number_of_RTTs_as_for_10_tables to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:123: success: expected GenerateObjects/generate_50000_tables to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:123: success: expected GenerateObjects/generate_100_tables_from_template to perform KV lookups in [11, 11], got 11
    validate_benchmark_data.go:123: success: expected ShowGrants/grant_2_roles to perform KV lookups in [14, 14], got 14
    validate_benchmark_data.go:123: success: expected ShowGrants/grant_3_roles to perform KV lookups in [16, 16], got 16
    validate_benchmark_data.go:123: success: expected AlterTableAddCheckConstraint/alter_table_add_1_check_constraint to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected AlterTableAddCheckConstraint/alter_table_add_2_check_constraints to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected DropDatabase/drop_database_0_tables to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:123: success: expected DropDatabase/drop_database_1_table to perform KV lookups in [18, 18], got 18
    validate_benchmark_data.go:123: success: expected DropDatabase/drop_database_2_tables to perform KV lookups in [18, 18], got 18
    validate_benchmark_data.go:123: success: expected VirtualTableQueries/select_crdb_internal.tables_with_1_fk to perform KV lookups in [1, 1], got 1
    validate_benchmark_data.go:123: success: expected VirtualTableQueries/select_crdb_internal.invalid_objects_with_1_fk to perform KV lookups in [1, 1], got 1
    validate_benchmark_data.go:123: success: expected VirtualTableQueries/virtual_table_cache_with_schema_change to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected DropView/drop_1_view to perform KV lookups in [16, 16], got 16
    validate_benchmark_data.go:123: success: expected DropView/drop_2_views to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:123: success: expected DropSequence/drop_1_sequence to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected DropSequence/drop_2_sequences to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:123: success: expected Truncate/truncate_1_column_0_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected Truncate/truncate_1_column_1_row to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected Truncate/truncate_1_column_2_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected Truncate/truncate_2_column_0_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected Truncate/truncate_2_column_1_rows to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:123: success: expected AlterTableDropConstraint/alter_table_drop_1_check_constraint to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected AlterTableDropConstraint/alter_table_drop_2_check_constraints to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:123: success: expected AlterTableConfigureZone/alter_table_configure_zone_5_replicas to perform KV lookups in [8, 8], got 8
    validate_benchmark_data.go:123: success: expected AlterTableConfigureZone/alter_table_configure_zone_7_replicas_ to perform KV lookups in [8, 8], got 8
    validate_benchmark_data.go:123: success: expected ORMQueries/pg_is_other_temp_schema to perform KV lookups in [6, 6], got 6
    validate_benchmark_data.go:123: success: expected RevokeRole/revoke_1_role to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:76: see the -rewrite flag to re-run the benchmarks and adjust the expectations
    validate_benchmark_data.go:77: usage: ./dev test --rewrite pkg/bench/rttanalysis -f=TestBenchmarkExpectation/ --test-args '-rewrite-iterations=N'
    validate_benchmark_data.go:110: -- test log scope end --
test logs left over in: outputs.zip/logTestBenchmarkExpectation308705980
--- FAIL: TestBenchmarkExpectation (1130.35s)
--- FAIL: TestBenchmarkExpectation (1130.57s)
=== RUN   TestBenchmarkExpectation/ORMQueries
    --- FAIL: TestBenchmarkExpectation/ORMQueries (1122.82s)
=== RUN   TestBenchmarkExpectation/ORMQueries/prisma_column_descriptions
    rtt_analysis_bench.go:206: ORMQueries/prisma_column_descriptions: got 167.5, expected 133
    rtt_analysis_bench.go:212: wrote jaeger trace to /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/testlogs/pkg/bench/rttanalysis/rttanalysis_test/shard_1_of_16_run_2_of_3/test.outputs/logTestBenchmarkExpectation308705980/ORMQueries_prisma_column_descriptions.jaeger.json
        --- FAIL: TestBenchmarkExpectation/ORMQueries/prisma_column_descriptions (713.04s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBenchmarkExpectation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/bench/rttanalysis/registry.go
+++ b/pkg/bench/rttanalysis/registry.go
@@ -62,6 +62,7 @@ func (r *Registry) RunExpectations(t *testing.T) {
 	skip.UnderStress(t)
 	skip.UnderRace(t)
 	skip.UnderShort(t)
+	skip.UnderDeadlock(t)
 
 	runBenchmarkExpectationTests(t, r)
 }

--- a/pkg/sql/backfill_protected_timestamp_test.go
+++ b/pkg/sql/backfill_protected_timestamp_test.go
@@ -42,6 +42,7 @@ func TestValidationWithProtectedTS(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	skip.UnderDeadlock(t, "test takes too long")
 	skip.UnderStress(t, "test takes too long")
 	skip.UnderRace(t, "test takes too long")
 

--- a/pkg/sql/schemachanger/sctest/backup.go
+++ b/pkg/sql/schemachanger/sctest/backup.go
@@ -41,6 +41,7 @@ func BackupSuccess(t *testing.T, path string, factory TestServerFactory) {
 	// These tests are expensive.
 	skip.UnderStress(t)
 	skip.UnderRace(t)
+	skip.UnderDeadlock(t)
 
 	cumulativeTestForEachPostCommitStage(t, path, factory, func(t *testing.T, cs CumulativeTestCaseSpec) {
 		backupSuccess(t, factory, cs)
@@ -53,6 +54,7 @@ func BackupRollbacks(t *testing.T, path string, factory TestServerFactory) {
 	// These tests are expensive.
 	skip.UnderStress(t)
 	skip.UnderRace(t)
+	skip.UnderDeadlock(t)
 	// These tests are only marginally more useful than BackupSuccess
 	// and at least as expensive to run.
 	skip.UnderShort(t)
@@ -68,6 +70,7 @@ func BackupSuccessMixedVersion(t *testing.T, path string, factory TestServerFact
 	// These tests are expensive.
 	skip.UnderStress(t)
 	skip.UnderRace(t)
+	skip.UnderDeadlock(t)
 	// These tests are only marginally more useful than BackupSuccess
 	// and at least as expensive to run.
 	skip.UnderShort(t)
@@ -84,6 +87,7 @@ func BackupRollbacksMixedVersion(t *testing.T, path string, factory TestServerFa
 	// These tests are expensive.
 	skip.UnderStress(t)
 	skip.UnderRace(t)
+	skip.UnderDeadlock(t)
 	// These tests are only marginally more useful than BackupSuccess
 	// and at least as expensive to run.
 	skip.UnderShort(t)

--- a/pkg/sql/ttl/ttljob/BUILD.bazel
+++ b/pkg/sql/ttl/ttljob/BUILD.bazel
@@ -95,6 +95,7 @@ go_test(
         "//pkg/sql/types",
         "//pkg/testutils",
         "//pkg/testutils/serverutils",
+        "//pkg/testutils/skip",
         "//pkg/testutils/sqlutils",
         "//pkg/testutils/testcluster",
         "//pkg/util/leaktest",

--- a/pkg/sql/ttl/ttljob/ttljob_test.go
+++ b/pkg/sql/ttl/ttljob/ttljob_test.go
@@ -40,6 +40,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/types"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
+	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
@@ -570,6 +571,10 @@ func TestRowLevelTTLJobMultipleNodes(t *testing.T) {
 func TestRowLevelTTLJobRandomEntries(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
+
+	// This test is very slow.
+	skip.UnderDeadlock(t)
+	skip.UnderRace(t)
 
 	rng, _ := randutil.NewTestRand()
 

